File: Hospital.php
<?php

namespace App\Model;

class Hospital
{
    public $tamaño = "";
    public $color = "";
    public $ubicacion = "";

    //nombre
    public function __construct($tam)
    {
       $this->establecerTamaño();
    }

    public function obtenerTamaño()
    {
        return $this->tamaño;
    }

    public function establecerTamaño($tam)
    {
        $this->tamaño = $tam;
    }

    //color
    public function __construct($col)
    {
       $this->establecerColor();
    }

    public function obtenerColor()
    {
        return $this->color;
    }

    public function estableceColor($col)
    {
        $this->color = $col;
    }

    //ubicacion
    public function __construct($ubi)
    {
       $this->establecerUbicacion();
    }

    public function obtenerUbicacion()
    {
        return $this->ubicacion;
    }

    public function establecerUbicacion($ubi)
    {
        $this->ubicacion = $ubi;
    }
}

File: Hospital.java
package Trabajo;

public class Hospital {

    public String color;

    public String tamaño;

    public String especialidad;

    public Hospital(String col, String tam, String esp)
    {
        establecerColor(col);
        establecerTamaño(tam);
        establecerEspecialidad(esp);
    }

  
    public String obtenerColor()
    {
        return color;
    }

    public void establecerColor(String col)
    {
        color = col;
    }

    //genero
    public String obtenerTamaño()
    {
        return tamaño;
    }

    public void establecerTamaño(String tam)
    {
        tamaño=tam;
    }

    //edad
    public String obtenerEspecialidad()
    {
        return especialidad;
    }
    public void establecerEspecialidad(String esp)
    {
        especialidad = esp;
    }
    
}
